### PR TITLE
Add tt:image test

### PR DESCRIPTION
**Pull request by palemieux**

### CODE REVIEW

File: imsc1_1/ttml/initial/initial001.ttml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nigelmegitt) [inline imsc1_1/ttml/initial/initial001.ttml]: Not sure the initial tests are supposed to be in this branch? They seem to be in a different pull request too.

**Comment by palemieux** [comment]
@nigelmegitt I have fixed the aspect ratio.

**Comment by palemieux** [comment]
@nigelmegitt Ping

**Comment by nigelmegitt** [comment]
@palemieux sorry, thanks for waking me up on this one. I just re-reviewed and I think there's still something unexpected here.

`tts:extent="640px 480px" and `ttp:displayAspectRatio="4 3"` but the output images are still 16:9, specifically 640x360px. I'm expecting 640x480px images for this test case.

I can see that the placed image is no longer squeezed, but the apparent location is wrong. It should have a top left at 128px 128px but by my rough measurements its top left is actually 176px 96px. Also its extent is supposed to be 160px 120px but I measure it as 120px 90px - the same aspect ratio but shrunk. The source image measures correctly as 160px 120px.

It seems as though two unexpected things have happened:
1. The reference output image is the wrong aspect ratio
2. The reference output image has been shrunk

I'm not expecting this test case to demonstrate positioning of a 4:3 root container region within a 16:9 context but that is what it appears to do. Unfortunately the reference image does not show any kind of boundary between the root container region and the outer context so that doesn't look very clear.

Your thoughts @palemieux?

**Comment by palemieux** [comment]
@nigelmegitt See improved test

**Comment by palemieux** [comment]
@nigelmegitt All tests are rendered to same resolution as described at https://github.com/w3c/imsc-tests/blob/master/imsc1/README.md
